Configure

```diff
@@ -1326,16 +1330,13 @@ if ($strict_warnings)
 	my $wopt;
 	die "ERROR --strict-warnings requires gcc or gcc-alike"
             unless defined($predefined{__GNUC__});
-	foreach $wopt (split /\s+/, $gcc_devteam_warn)
+	foreach $wopt (split /\s+/, $devteam_common_warn)
 		{
 		$config{cflags} .= " $wopt" unless ($config{cflags} =~ /(?:^|\s)$wopt(?:\s|$)/)
 		}
-	if (defined($predefined{__clang__}))
+	foreach $wopt (split /\s+/, $devteam_additional_warn{$ecc})
```

Comment:
But there is no $ecc anymore...

Reply:
Damnit.  That...  unfortunate.  Let me see what I can arrange